contactSection.html

```diff
@@ -0,0 +1,14 @@
+		<section class="contactMe">
```

Comment:
is this still meant to be part of the build, if not best practice to remove it

Reply:
Have removed the file